google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/SpannerPool.java

```diff
@@ -156,6 +156,7 @@ static class SpannerPoolKey {
     private final String userAgent;
     private final String databaseRole;
     private final boolean routeToLeader;
+    private final boolean enableEndToEndTracing;
```

Comment:
nit: group with the other tracing properties

Reply:
Done